multiversx_sdk/network_providers/http_resources.py

```diff
@@ -109,19 +110,24 @@ def _transaction_from_network_response(tx_hash: str, response: dict[str, Any]) -
 
 
 def transaction_logs_from_response(raw_response: dict[str, Any]) -> TransactionLogs:
-    address = raw_response.get("address", "")
+    address = raw_response.get("address", "") or EmptyAddress()
```

Comment:
In this particular context, checking the type is somehow exotic. Maybe (separate function, as Dana said):

```
def convert_bech32_to_address(value: str):
    if not address:
        return EmptyAddress()
    return Address.new_from_bech32(address)
    
# Usage
address = convert_bech32_to_address(raw_response.get("address", ""))
```

Reply:
extracted a separate function